src/buildblock/recon_array_functions.cxx

```diff
@@ -224,48 +225,50 @@ void divide_and_truncate(Viewgram<float>& numerator,
       };
 
 #else
-      if(b<bs+rim_truncation_sino ||
-	 b>be-rim_truncation_sino ) 
-	{
-	  numerator[r][b] = 0;
-	}
+      if(b<bs+rim_truncation_sino || b>be-rim_truncation_sino )
+	    numerator[r][b] = 0;
       else
-	{
-	  float& num = numerator[r][b];
-	  if (num<=small_value) // KT Feb2011 was "num<small_value", resulting in a BUG if the whole numerator viewgram was zero
-	    { 
-	      // we think num was really 0 
-	      // (we compare with small_value due to rounding errors)
-	      // this case includes 0/0, but also num<0	     
-	      num = 0;
-	      if (num<0) count2++;
-	    }
-	  else
-	    {
-	      const float max_quotient = 10000.F;
-	      const float denom = denominator[r][b];
-	      // set quotient to min(numerator/denominator, max_quotient)
-	      // a bit tricky to avoid division by 0	  
-	      // we do this by effectively using
-	      // new_denom = max(denominator[r][b], max_quotient/num)
-	      // Note that this includes the case if a negative denominator
-	      // (in case somebody forward projects an image with negatives)
-	      if (num > max_quotient*denom)
-		{
-		  // cancel singularity
-		  count++;
-		  if (log_likelihood_ptr != NULL) 
-		    sub_result -= double(num*log(num/max_quotient));
-		  num = max_quotient;
-		}
-	      else
-		{
-		  if (log_likelihood_ptr != NULL) 
-		    sub_result -= double(num*log(denom));
-		  num = num/denom;
-		}
-	    }
-	}
+      {
```

Comment:
a lot of this is just reformatting as well. I can see why, but this code is at the heart of a lot of things. I'd rather not change it unless it needs to